### PR TITLE
add recipe for VisiData

### PR DESCRIPTION
**Pull request by anjakefala**

### CODE REVIEW

**Comment by anjakefala** [comment]
Ack. This is my first conda recipe. I see the failures and I will start addressing them tomorrow. =)

**Comment by anjakefala** [comment]
Ping @conda-forge/staged-recipes

**Comment by anjakefala** [comment]
Ping @conda-forge/staged-recipes :)

**Comment by scopatz** [comment]
Thanks!

File: recipes/visidata/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: True  # [py<34]
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python >=3.4
```
Comment (isuruf) [inline recipes/visidata/meta.yaml]: I'm commenting here because it's easier. This is wrong. You should remove `>=3.4` from here and keep the skip.

Comment (anjakefala) [reply recipes/visidata/meta.yaml]: :+1: Will do so now.

Comment (anjakefala) [reply recipes/visidata/meta.yaml]: I should remove it from the `run` requirements as well?

Comment (isuruf) [reply recipes/visidata/meta.yaml]: Yes

**Pull request merged**